online-shop/src/app/products.service.ts

```diff
@@ -0,0 +1,67 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {Observable} from "rxjs";
+import {FormGroup} from "@angular/forms";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductsService {
+
+  readonly ROOT_URL = "http://localhost:3000"
```

Comment:
**suggestion:** you cand keep this url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)